mesheryctl/internal/cli/root/system/dashboard.go

```diff
@@ -71,6 +72,9 @@ mesheryctl system dashboard
 // Open Meshery UI in browser and use port-forwarding (if default port is taken already)
 mesheryctl system dashboard --port-forward
 
```

Comment:
Please clarify whether this port number assignment is by default or just an example. Why not use 9081 as the default (and as the example)?